src/lib/y2packager/product.rb

```diff
@@ -47,6 +47,15 @@ def self.available_base_products
       Y2Packager::ProductReader.available_base_products
     end
 
+    # Returns the selected base product
+    #
+    # It assumes that at most 1 product could be selected.
+    #
+    # @return [Y2Packager::Product] Selected base product
+    def self.selected_base
+      available_base_products.find(&:selected?)
```

Comment:
I have a feeling that sooner or later we might need a separate class just for holding the base product selected by the user. With many extensions available and many derived products (like SLES4SAP, OES) and possible dependencies/conflicts the assumption of only 1 selected base product might not be valid anymore...

But that's something for later...